imports/services/BoardService.ts

```diff
@@ -79,4 +83,35 @@ export class BoardService extends BaseService {
     }
     return board
   }
+
+  public async starBoard(boardId: string): Promise<boolean> {
+    const userId = Meteor.userId()
+    if (userId != null) {
+      return BoardFavoritesCollection.addFavoriteBoard(userId, boardId)
+    } else {
+      throw new Meteor.Error('Cannot Star the board')
```

Comment:
error message is not vaggue, you can use message like "Unauthorized" or "Permission Denied" to indicate that this user is not logged in.